core/src/test/java/org/apache/calcite/test/DiffRepository.java

```diff
@@ -765,7 +765,7 @@ public static synchronized DiffRepository lookup(
       final File logFile =
           new File(
               Sources.of(refFile).file().getAbsolutePath()
-                  .replace("test-classes", "surefire"));
+                  .replace(".xml", "_surefire.xml"));
```

Comment:
what is surefire?

Reply:
hi @vlsi , I'm not sure I fully understand your question. I try to explain the logic of this PR.
I think the original intention was to create a log file in "surefire" directory, containing the actual relnode tree for sql after run test.
But now, the ref file is not in "test-classes" directory (no test-classes directory exist), make log file and ref file become the same file. So update log file will update ref file too.
This pr just to make sure log file and ref file are different files.

Reply:
I suggest we use a human-understandable naming.
For instance: `${name}.actual.xml`. Is it the file with the actual output? WDYT?

Frankly speaking, it is puzzling why `xml file` is called a log file, but if it works who cares.

Reply:
We should at least add a check here that the charSequence to replace must exists, or, the log file path and ref file path must be different, to avoid such problems when we do later code upgrade.

Reply:
@danny0405 , for this case ".xml" is sure to exists, the code `final URL refFile = findFile(clazz, ".xml");` guarantee that. In case code upgrade in future, I'll add an check to make sure re file and log file are different.

@vlsi I'm OK with both names. I'll update the PR as you suggested.